test/_linux.py

```diff
@@ -472,6 +480,73 @@ def test_procfs_path(self):
             psutil.PROCFS_PATH = "/proc"
             os.rmdir(tdir)
 
+    def test_psutil_is_reloadable(self):
+        imp.reload(psutil)
+
+    def test_no_procfs_for_import(self):
+        my_procfs = tempfile.mkdtemp()
+
+        with open(os.path.join(my_procfs, 'stat'), 'w') as f:
+            f.write('cpu   0 0 0 0 0 0 0 0 0 0\n')
+            f.write('cpu0  0 0 0 0 0 0 0 0 0 0\n')
+            f.write('cpu1  0 0 0 0 0 0 0 0 0 0\n')
+
+        self.assertNotAlmostEqual(psutil.cpu_percent(), 0)
+        self.assertNotAlmostEqual(sum(psutil.cpu_times_percent()), 0)
```

Comment:
What are you trying to test here? Why did you use `assertNotAlmostEqual` and not `assertNotEqual`?